webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -187,14 +198,25 @@ class ManageManifestModal extends Component {
                     />
 
                     <TooltipButton
-                      onClick={this.deleteManifest}
+                      onClick={() => this.showDeleteManifestModal(true)}
                       tooltipId="delete-manifest-button-tooltip"
                       tooltipText={__('This is disabled because a manifest task is in progress.')}
                       tooltipPlacement="top"
                       title={__('Delete')}
                       disabled={taskInProgress}
                     />
 
+                    <ConfirmDialog
+                      show={this.state.showDeleteManifestModalDialog}
+                      title={__('Confirm delete manifest')}
+                      dangerouslySetInnerHTML={{
+                        __html: sprintf(__(DeleteManifestModalText)),
```

Comment:
I'm afraid this is not going to work. AFAIK code isn't evaluated when the translations are being extracted.

Reply:
BTW `sprintf` is useless here.